casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperMergeSpec.scala

```diff
@@ -46,39 +46,6 @@ class MultiParentCasperMergeSpec extends FlatSpec with Matchers with Inspectors
     }
   }
 
-  // TODO enable tests below
```

Comment:
this does not make sense in presence of new MergeSpec. We can fix it but it'll require changes to some MergeSpec API (`merges(lookup, lookup, setup)`) is not present anymore. I don't think it worth it.

Reply:
Do we have test with stdout and registry lookup when checking *all mergeability rules*?